### PR TITLE
Dora metrics

### PR DESCRIPTION
**Pull request by mk-armah**
# Description

## Updated docs pages
Added rate limiting and backoff strategy to `deployment_fequency.py`

### CODE REVIEW

**Comment by mk-armah** [comment]
@MPTG94

File: docs/build-your-software-catalog/custom-integration/api/ci-cd/github-workflow/guides/dora-metrics.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-sidebar_position: 1
+sidebar_position: 10
 ---
 
 import PortTooltip from "/src/components/tooltip/tooltip.jsx";
 
 # DORA Metrics
 
-In this guide, we will create a GitHub action that computes the DORA Metrics for a service (repository) on schedule and ingests the results to Port.
+In this guide, we will create a github action that computes the DORA Metrics for a service (repository) on schedule and ingests the results to Port.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/custom-integration/api/ci-cd/github-workflow/guides/dora-metrics.md:9]:
```suggestion
In this guide, we will create a GitHub action that computes the DORA Metrics for a service (repository) on schedule and ingests the results to Port.

```

File: docs/build-your-software-catalog/custom-integration/api/ci-cd/github-workflow/guides/dora-metrics.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
 ## Prerequisites
 1. A GitHub repository in which you can trigger a workflow that we will use in this guide.
-2. A blueprint in Port to host the Dora Metrics.
+2. A blueprint in port to host the Dora Metrics.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/custom-integration/api/ci-cd/github-workflow/guides/dora-metrics.md:13]:
```suggestion
2. A blueprint in Port to host the Dora Metrics.

```

File: docs/build-your-software-catalog/custom-integration/api/ci-cd/github-workflow/guides/dora-metrics.md [hunk 3]
```diff
@@ -141,7 +141,7 @@ Below, you can find the JSON for the `DORA Metrics` blueprint required for the g
 
 Follow these steps to get started:
 
-1. Create the following GitHub action secrets:
+1. Create the following GitHub Action secrets:
```
Comment (hadar-co) [inline docs/build-your-software-catalog/custom-integration/api/ci-cd/github-workflow/guides/dora-metrics.md:144]:
```suggestion
1. Create the following GitHub action secrets:

```

File: docs/build-your-software-catalog/custom-integration/api/ci-cd/github-workflow/guides/dora-metrics.md [hunk 4]
```diff
@@ -880,4 +907,4 @@ if __name__ == "__main__":
 ```
 </details>
 
-Congrats 🎉 You've successfully scheduled a GitHub action to periodically ingest estimated `DORA Metrics` for GitHub repository(s).
+Congrats 🎉 You've successfully scheduled a github action to periodically ingest estimated `DORA Metrics` for github repository(s).
```
Comment (hadar-co) [inline docs/build-your-software-catalog/custom-integration/api/ci-cd/github-workflow/guides/dora-metrics.md:910]:
```suggestion
Congrats 🎉 You've successfully scheduled a GitHub action to periodically ingest estimated `DORA Metrics` for GitHub repository(s).

```

**Review by hadar-co — APPROVED** [review]

**Pull request merged**